schema/dataKeys.json

```diff
@@ -0,0 +1,352 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "fbc": {
+      "type": "object",
+      "properties": {
+        "request": {
+          "type": "string",
+          "description": "The internal pipline name to handle requests e.g. iib"
+        },
+        "fbcPublishingCredentials": {
+          "type": "string",
+          "description": ""
+        },
+        "buildTags": {
+          "type": "array",
+          "description": "",
+          "items": {
+            "type": "string"
+          }
+        },
+        "addArches": {
+          "type": "array",
+          "description": "",
+          "items": {
+            "type": "string"
+          }
+        },
+        "hotfix": {
+          "type": "boolean",
+          "description": "Indicates if the build is a hotfix"
+        },
+        "preGA": {
+          "type": "boolean",
+          "description": "Indicates if the build is a preGA"
+        },
+        "stagedIndex": {
+          "type": "boolean",
+          "description": "Indicates if the build is a staged index"
+        },
+        "productName": {
+          "type": "string",
+          "description": "The product name e.g. Red Hat Product "
+        },
+        "productVersion": {
+          "type": "string",
+          "description": "The product version e.g v1.0.0"
+        },
+        "buildTimeoutSeconds": {
+          "type": "integer",
+          "description": "The build timeout in seconds e.g. 1500"
+        },
+        "requestTimeoutSeconds": {
+          "type": "integer",
+          "description": "The requested timeout in seconds e.g. 1500"
+        },
+        "timestampFormat": {
+          "type": "string",
+          "description": ""
+        },
+        "issueId": {
+          "type": "string",
+          "description": "The issue ID e.g. bz123456"
+        },
+        "allowedPackages": {
+          "type": "array",
+          "description": "The allowed packages e.g. ['example-package-1'], ['example-package-2'] ",
+          "items": {
+            "type": "string"
+          }
+        },
+        "fromIndex": {
+          "type": "string",
+          "description": "The source index image e.g. registry-proxy.engineering.redhat.com/rh-osbs/iib-preview-rhtap:v4.09  "
+        },
+        "targetIndex": {
+          "type": "string",
+          "description": "The index image e.g. quay.io/redhat/redhat----preview-operator-index:v4.10 "
+        },
+        "binaryImage": {
+          "type": "string",
+          "description": "The OCP binary image to be baked into the index image e.g. registry.redhat.io/openshift4/ose-operator-registry:v4.11"
+        },
+        "pipelineImage": {
```

Comment:
I have to search for this one. Although it can be set here , the best should have it only in the `sign` section.

Reply:
I have removed it from `.fbc.`